documentation/running_bolt_commands.md

```diff
@@ -1,418 +1,640 @@
-    ```
-    bolt command run --targets '@targets.txt'
-    ```
+- _PowerShell cmdlet_
 
--   To pass targets on `stdin`, on the command line, use a command to generate a
-    target list, and pipe the result to Bolt with `-` after `--targets`:
-    ```
-    <COMMAND> | bolt command run --targets -
-    ```
+  Use the `-SaveRerun` argument with a value of `$false` to disable saving the
+  rerun file:
 
-    For example, if you have a target list in a text file:
-    ```
-    cat targets.txt | bolt command run --targets -
-    ```
+  ```powershell
+  Invoke-BoltTask -Name restart_server -Targets servers -Rerun success -SaveRerun:$false
+  ```
 
--   To pass targets as IP addresses, use `protocol://user:password@host:port` or
-    inventory group name. You can use a domain name or IP address for `host`,
-    which is required. Other parameters are optional.
-    ```
-    bolt command run --targets ssh://user:password@[fe80::34eb:ff1:b584:d7c0]:22,
-    ssh://root:password@hostname, pcp://host01, winrm://Administrator:password@hostname
-    ```
+### Specifying connection credentials
 
+To establish connections with remote targets, Bolt will need to provide
+credentials to the target. These credentials can be provided at the command
+line or in an inventory file, and the credentials you provide may vary based
+on the operating system the target is running.
 
-#### Specify targets from an inventory file
+Whether a target is running a Unix-like operating system or Windows, the
+simplest way to specify credentials is to pass the `user` and `password`
+to the Bolt command:
 
-To specify targets from an inventory file, reference targets by target name, a
-glob matching names in the file, or the name of a group of targets.
--   To match all targets in both groups listed in the inventory file example:
-    ```
-    --targets elastic_search,web_app
-    ```
--   To match all the targets that start with "elasticsearch" in the inventory
-    file example:
+- _\*nix shell command_
+
+  ```shell
+  bolt command run 'pwd' --targets servers --user bolt --password puppet
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltCommand -Command 'pwd' -Targets servers -User bolt -Password puppet
+  ```
+
+If you would prefer to have Bolt securely prompt for a password, so that it
+does not appear in a process listing or on the console, use the
+`password-prompt` option instead:
+
+- _\*nix shell command_
+
+  ```shell
+  bolt command run 'pwd' --targets servers --user bolt --password-prompt
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltCommand -Command 'pwd' -Targets servers -User bolt -PasswordPrompt
+  ```
+
+### Specifying a transport
+
+When Bolt establishes a connection with a target, it uses a specific transport
+to do so. By default, Bolt will connect to a target using the `ssh` transport.
+However, you can specify a specific transport that Bolt should use to connect
+to each target.
```

Comment:
```suggestion
Bolt uses a specific transport to establish a connection with a target. 
By default, Bolt connects to targets using the `ssh` transport. You can 
use one of the methods below to set a different transport from the 
command line, or you can configure transports in your inventory file.
```
This gets the inventory file bit out of the way early so you don't have to talk about it later